CONTRIBUTING.md

```diff
@@ -0,0 +1,105 @@
+# Video Headline - Contributing
+
+## Table of Contents
+
+- [Running the application in local environment](#running-the-application-in-local-environment)
+- [How to build the player](#how-to-build-the-player)
+- [How to build the docker](#how-to-build-the-docker)
+- [How to run the tests](#how-to-run-the-tests)
+- [How to use the linters](#how-to-use-the-linters)
+
+### Running the application in local environment
+
+To set up the project locally, follow the instructions provided below.
+
+#### Prerequisites
+
+- AWS Account: Necessary for hosting and delivering video content.
+- Docker and Docker compose: Video Headline runs inside Docker containers, so it is necessary to have Docker and Docker Compose installed.
+- Yarn and Node.js(v10): Required to deploy AWS configurations and build the playerReact component.
+- Python: Necessary for running Django and other Python-based tools.
+- AWS CLI: Useful for configuring and managing AWS services from the command line.
+
+#### Create .env file
+
+Create a .env file at the root of the project with all the variables defined in the .env-example file and their respective values.
```

Comment:
i think we should explain all of them, and their default values in case user did not change anything in regards to config (for example, DATABASE_USER will always be "qualabs" if developer does not change anything in docker-commands/db_init/init.sh)